src/internal/converters/ics/ics.go

```diff
@@ -481,8 +488,16 @@ func updateEventProperties(ctx context.Context, event models.Eventable, iCalEven
 			}
 		}
 
+		// It is possible that we get non email items like the below
+		// one which is an internal representation of the user in the
+		// Exchange system. While we can technically output this as an
+		// attendee, it is not useful plus other downstream tools like
+		// ones to use PST can choke on this.
+		// /o=ExchangeLabs/ou=ExchangeAdministrative Group(FY...LT)/cn=Recipients/cn=883...4a-John Doe
```

Comment:
Does it make sense to log the address which does not qualify as an email? This might be helpful to understand the data we're working with.